08-projects.Rmd

```diff
@@ -2649,23 +2649,7 @@ https://thedatamine.github.io/the-examples-book
 
 It is highly recommended to read through, search, and explore these examples to help solve problems in this project.
 
-**Important note:** It is highly recommended that you use https://rstudio.scholar.rcac.purdue.edu/. Simply click on the link and login using your Purdue account credentials. Use another system at your own risk. The version of RStudio on https://desktop.scholar.rcac.purdue.edu/ (which uses ThinLinc), is 99.9.9, and is known to have some strange issues when running code chunks.
-
-Don't forget the very useful documentation shortcut `?`. To use, simply type `?` in the console, followed by the name of the function you are interested in. 
```

Comment:
This and the following paragraphs are modified as this project is mainly about `bash` code.